scalafix-cli/src/main/scala/scalafix/internal/v1/Args.scala

```diff
@@ -60,7 +60,7 @@ case class Args(
       "default Scala2" +
```

Comment:
I guess this is no longer useful since the default shows up in the help message?

Reply:
ok